x-pack/plugins/enterprise_search/server/utils/create_ml_inference_pipeline.ts

```diff
@@ -74,7 +74,7 @@ export const createMlInferencePipeline = async (
   destinationField: string,
   esClient: ElasticsearchClient
 ): Promise<CreatedPipeline> => {
-  const inferencePipelineGeneratedName = `ml-inference-${pipelineName}`;
```

Comment:
I think you'll get a merge conflict with [this change](https://github.com/elastic/kibana/pull/141015/files#diff-5e7b6792758030bf960d19769927afa99b0298dfe093d7b87f2a5b1a64598dfcL77).